src/common/updating/installer.h

```diff
@@ -5,7 +5,7 @@
 #include <future>
 
 #include "notifications.h"
-#include "../VersionHelper.h"
+#include "../version/helper.h"
```

Comment:
In some places we have
```
#include "../version/helper.h"
```
in others
```
#include <common/version/helper.h>
```
Same for `#include "../version/version.h"`.
Can we use the second notation everywhere?